src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1085,6 +1095,27 @@ public WrapModeType WrapModeV
             }
         }
 
+        /// <summary>
+        /// Gets or sets the mode to adjust view size to preserve the aspect ratio of the image resource.
+        /// If this is set to be true, then the width or height, which is not set by user explicitly, can be adjusted to preserve the aspect ratio of the image resource.
+        /// AdjustViewSize works only if ImageView is added to a View having Layout.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AdjustViewSize
```

Comment:
Bindable property로 하는것이 아니면, private변수는 삭제하고 auto generate된 property구현을 사용해도 될것 같습니다.
```suggestion
        public bool AdjustViewSize {get; set;}
```

Reply:
리뷰 감사합니다.
코멘트주신대로 수정하였습니다.